test/hotspot/jtreg/compiler/startup/StartupOutput.java

```diff
@@ -60,8 +61,9 @@ public static void main(String[] args) throws Exception {
             throw new Exception("VM crashed with exit code " + exitCode);
         }
 
+        int minInitialSize = 800 + (Platform.isS390x() ? 800 : 0);
```

Comment:
Do you unconditionally need the extra space or maybe only for fast debug builds?
Maybe add a comment on which initialization code sizes (interpreter, stubs, ...) you observe.